proto/internal/temporal/server/api/persistenceblobs/v1/message.proto

```diff
@@ -295,9 +295,9 @@ message WorkflowExecutionInfo {
     bytes event_branch_token = 45;
     int64 signal_count = 46;
     int64 history_size = 47 [deprecated = true];
-    string client_library_version = 48;
-    string client_feature_version = 49;
-    string client_impl = 50;
+    reserved 48;
+    reserved 49;
+    reserved 50;
```

Comment:
This info is not persisted anymore as it is not used by code. @samarabbas, I am wondering if we still want to persist client name and version for debugging purpose.